vm/vm/src/actions/runCall/RunCallAction.ts

```diff
@@ -4,7 +4,7 @@ import { type Address, type Hex } from 'viem'
  * Tevm action to execute a call on the vm
  */
 export type RunCallAction = {
-	to: Address
+	to?: Address
```

Comment:
This is a good change but there is also a thing called CallActionValidator that also needs to be updated